beacon-chain/core/state/transition.go

```diff
@@ -563,7 +569,9 @@ func verifyOperationLengths(state *stateTrie.BeaconState, body *ethpb.BeaconBloc
 			params.BeaconConfig().MaxVoluntaryExits,
 		)
 	}
-
+	if state.Eth1Data() == nil {
```

Comment:
This makes a copy of the field accessed, can you assign the result and reuse it on the next if statement so we aren't making multiple copies of the data?

Reply:
thanks!
done